apps/backend/src/app/api/admin/bookings/[id]/route.test.ts

```diff
@@ -0,0 +1,84 @@
+import { AUTH_COOKIE_NAME } from "@repo/shared"
+import { getReasonPhrase, StatusCodes } from "http-status-codes"
+import { cookies } from "next/headers"
+import BookingDataService from "@/data-layer/services/BookingDataService"
+import { createMockNextRequest } from "@/test-config/backend-utils"
+import { bookingCreateMock } from "@/test-config/mocks/Booking.mock"
+import { adminToken, casualToken, memberToken } from "@/test-config/vitest.setup"
+import { DELETE } from "./route"
+
+describe("/api/admin/bookings/[id]", async () => {
+  const cookieStore = await cookies()
+  const bookingDataService = new BookingDataService()
+
+  describe("DELETE", () => {
+    it("should return 401 if user is a casual", async () => {
+      cookieStore.set(AUTH_COOKIE_NAME, casualToken)
+      const booking = await bookingDataService.createBooking(bookingCreateMock)
+      const response = await DELETE(
+        createMockNextRequest(`/api/admin/bookings/${booking.id}`, "DELETE"),
+        {
+          params: Promise.resolve({ id: booking.id }),
```

Comment:
you dont need to create a booking if testing for 401

```suggestion
    it("should return 401 if user is a casual", async () => {
      cookieStore.set(AUTH_COOKIE_NAME, casualToken)
      const response = await DELETE(
        createMockNextRequest("", "DELETE"),
        {
          params: Promise.resolve({ id: "test" }),
```